ibis/backends/dask/tests/test_udf.py

```diff
@@ -432,6 +432,19 @@ def test_array_return_type_reduction_window(con, t, df, qs):
     tm.assert_series_equal(result, expected)
 
 
+def test_array_return_type_reduction_group_by(con, t, df, qs):
+    expr = t.groupby(t.key).aggregate(
```

Comment:
can you add the issue reference as a comment, also a 1-liner to explain the issue

Reply:
done!